forge-gui/res/cardsfolder/upcoming/desynchronization.txt

```diff
@@ -0,0 +1,5 @@
+Name:Desynchronization
+ManaCost:2 U U
+Types:Instant
+A:SP$ ChangeZoneAll | ChangeType$ Permanent.nonLand+nonHistoric | Origin$ Battlefield | Destination$ Hand | SpellDescription$ Return each nonland permanent that's not historic to its owner's hand. (Artifacts, legendaries, and Sagas are historic.)
```

Comment:
Historic is no real type, does that valid really work?

Reply:
I thought that non simply negates what comes after, it's used as nonToken etc. If it doesn't work with historic, it probably should. It could be, of course, split to explicitly negate each of Historic's components, but that seemed less elegant.